apps/docs/src/app/core/component-docs/dialog/examples/auto-label/auto-label-dialog-example.component.html

```diff
@@ -14,13 +14,15 @@ <h1 id="fd-dialog-header-10" fd-title fd-dialog-title>The History of Pineapple</
                 label="Interested"
                 [compact]="true"
                 (click)="dialog.close('Continue')"
+				ariaLabel="Interested emphasized"
             ></fd-button-bar>
 
             <fd-button-bar
                 fdInitialFocus
                 label="Cancel"
                 [compact]="true"
                 (click)="dialog.dismiss('Cancel')"
+				ariaLabel="Cancel transparent"
```

Comment:
is the user really interested in the fact that the button is transparent or emphasized?

Reply:
yes, it has been raised as issues by the accessibility team. the changes has been introduced post verifying with the team.

Reply:
in that case we should handle this from library itself, right? where `aria-label` can be set to the `ariaLabel` value and the button's `type`, so that user is not expected to enter button type as well.

Reply:
I am not sure how this satisfy any accessibility requirement to say the type of the button(e.g. transparent, ghost, default). @Lokanathan-k can you ask which ACC requirement is covering this?

Reply:
> yes, it has been raised as issues by the accessibility team. the changes has been introduced post verifying with the team.

I am not sure how this satisfy any accessibility requirement to say the type of the button(e.g. transparent, ghost, default). @Lokanathan-k can you ask which ACC requirement is covering this?

Reply:
@droshev the requirement from ACC team is to diffrentate the normal button and the emphasied button. and ACC team requires the fdtype="emphasised" to be read, and the chagne now has been introduced to handle it from arial label.